### ISSUE TITLE
508-defect-1 [SCREENREADER] FAQ needs to have aria-label to properly pronounce it

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#20237]
# [508-defect-1](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Noah Gelman

## Details

Text on page that says FAQ is not announced correctly. Screen readers pronounce it is "F*ck" (profanity). This occurs multiple times on the page.

## Acceptance Criteria

- [ ] Wrap FAQ text in a span with aria-label to correctly announce it as "Frequently Asked Questions"

## Proposed Solution (if known)

`<span aria-label="Frequently Asked Questions">FAQ</span>`

**Comment by RLHecht** [comment]
@noahgelman I think this would be a sitewide issue as well. @brianalloyd is this something that can done in the FE since we generally don't use source code when we add content within Drupal? Not sure what the best way to handle this is.

**Comment by brianalloyd** [comment]
Thanks @RLHecht we're scrubbing into this issue on FE.

**Comment by brianalloyd** [comment]
@kelsonic looking into sitewide impact.

**Comment by kelsonic** [comment]
Closing as deemed not an issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/20238#issuecomment-860942131 Reopen if needed! 🙂
